.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug
+about: Report a problem with an integration or a general feature.
```

Comment:
I they we can get away by saying 
`Report a problem with a general feature or the extension.`

and not mentioning integration here